mambaSharedFramework/Pantos-Generic HLS Playlist Parsing/Pantos-Generic Tag Validators/EXT_X_SESSION_DATAPlaylistValidator.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  EXT_X_SESSION_DATAPlaylistValidator.swift
+//  mamba
+//
+//  Created by Robert Galluccio on 8/31/24.
+//  Copyright © 2024 Comcast Corporation.
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License. All rights reserved.
+//
+
+import Foundation
+
+class EXT_X_SESSION_DATAPlaylistValidator: HLSPlaylistValidator {
+    static func validate(hlsPlaylist: any HLSPlaylistInterface) -> [HLSValidationIssue]? {
+        var issues = [HLSValidationIssue]()
+
+        let issue = duplicateIssue(tags: hlsPlaylist.tags.filter { $0.tagDescriptor == PantosTag.EXT_X_SESSION_DATA })
+        if let issue {
```

Comment:
I think you can combine this with above line `if let issue = duplicateIssue(......`

Reply:
Yeah... I've got a line on Xcode at 120 characters so was just trying to avoid going over that limit... The other way of avoiding it was going for something like:
```swift
if let issue = duplicateIssue(
    tags: hlsPlaylist.tags.filter { $0.tagDescriptor == PantosTag.EXT_X_SESSION_DATA }
) {
    // etc.
}
```
Not sure which one looks better... Or I could just let the line go long (since we don't have a linter in this project either).

Reply:
Updated here: f23275d35b6e50ab7578dc991b853d6d4255ec81

What do you think?

Reply:
I think it looks good